src/IFedoraClient.php

```diff
@@ -133,6 +133,16 @@ public function modifyResource($uri,
      */
     public function deleteResource($uri,
                                    $transaction = "");
+    /**
+     * Issues a COPY request to Fedora.
+     *
+     * @param string    $uri            Resource URI
+     * @param array     $destination    Destination URI
+     *
+     * @return string   
+     */
+    public function copyResource($uri,
+                                 $destination);
```

Comment:
I'd add transactionality here too.  Might want to do a lot of copying inside a transaction.